lightning/src/onion_message/messenger.rs

```diff
@@ -417,6 +417,22 @@ pub struct PendingOnionMessage<T: OnionMessageContents> {
 	pub reply_path: Option<BlindedMessagePath>,
 }
 
+/// A placeholder value used for paths within the codebase.
+///
+/// PATHS_PLACEHOLDER` is temporarily used as a default value in situations
+/// where a path index is required but has not yet been assigned or initialized.
+pub const PATHS_PLACEHOLDER: usize = 3;
```

Comment:
Rather than using this globally, we should use a different constant that makes sense wherever this is required. For example, this is used in `handle_onion_message_response` for a reply path even though only one path can ever be used.

Reply:
Makes sense!
Removed the global variable and instead introduced a default constructor for function using the default number of paths in [pr3246.06](https://github.com/shaavan/rust-lightning/commits/pr3246.06)